content/wicys.md

```diff
@@ -0,0 +1,16 @@
+---
```

Comment:
additionally, lets find a good picture for their banner/background on the page -- ask the wicys folks?

Reply:
Creating issue for this